modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/BaseDisplayTag.java

```diff
@@ -133,10 +137,6 @@ public String getRandomNamespace() {
 		return _randomNamespace;
 	}
 
-	public List<Object> getSelectedItems() {
```

Comment:
I'm ok to remove this, if there are no more usages in DXP. But as this is a breaking change, on a code not behind FF, I believe we should make a special commit message here, https://liferay.atlassian.net/wiki/spaces/ENGR/pages/2489483265/Process+to+register+breaking+changes+in+commits